content/general/concepts/http/http.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: 'HTTP'
+Description: 'HTTP is used for fetching HTML documents and other Web-based resources.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Browsers'
+  - 'Development'
+  - 'Errors'
+  - 'HTTP'
+CatalogContent:
+  - 'learn-intermediate-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Hypertext Transfer Protocol (HTTP) is used for fetching [HTML](https://www.codecademy.com/resources/docs/html) documents and other Web-based resources. It follows a model where a client (most commonly a browser) requests content stored somewhere on a server.
+
+## Caches and Proxies
+
+HTTP can be used for improving web performance with [caches](https://www.codecademy.com/resources/docs/general/cache) and [proxy servers](https://www.codecademy.com/resources/docs/general/proxy-server).
+
+Client browsers can use caches for serving content instead of making repeated requests for the same content. Examples of caches include:
+
+- [CDNs](https://www.codecademy.com/resources/docs/general/cdn) that retain copies of web content and serve from close network connections.
+- Proxy browser caches, like ones used in [progressive web apps](https://www.codecademy.com/resources/docs/general/progressive-web-application) that allow a single user to cache and access content offline.
+- Shared proxy caches that store resources for multiple users (internet service provider, company staff network, etc.).
+
+Proxies are used to mask a client's IP address by assigning one to a proxy server and have requests sent from there.
+
+## HTTPS
+
+Instead of HTTP, a more secure version, HTTPS, should be used in order to encrypt information sent between the client and server. This encryption is done with Transport Layer Security (or TLS, formerly [SSL](https://www.codecademy.com/resources/docs/general/ssl)). Some benifits of HTTPS include:
+
+- Financial data, like credit card information or bank account numbers, is protected from interception.
+- Domain-ownership can be better verified by users.
```

Comment:
Sure! Maybe a brief explanation of what they are/do with regards to HTTPS?

Reply:
That makes sense, since they're a significant part of how HTTPS works.

Reply:
This change may take me until tomorrow (because I wanna get it right 💪) but is [this government site about certs](https://https.cio.gov/certificates/) a good place to start, you think?